src/jllvm/vm/JIT.cpp

```diff
@@ -162,7 +162,6 @@ jllvm::JIT jllvm::JIT::create(ClassLoader& classLoader, GarbageCollector& gc, vo
     epciu->createLazyCallThroughManager(*es, llvm::pointerToJITTargetAddress(+[]
                                                                              {
                                                                                  // TODO: throw UnsatisfiedLinkError
```

Comment:
I think this TODO can be removed entirely since its properly implemented in the JNI layer

Reply:
The TODO or the whole statement?

Reply:
Just the TODO comment. The fatal error reporting is still useful during development.